components/segmented/segmented.spec.ts

```diff
@@ -177,16 +193,26 @@ describe('nz-segmented', () => {
 
     beforeEach(() => {
       fixture = TestBed.createComponent(NzSegmentedIconOnlyTestComponent);
+      fixture.detectChanges();
     });
 
-    it('should render only one element in .ant-segmented-item-label if the item is icon-only', () => {
-      const items = fixture.debugElement.queryAll(By.css('.ant-segmented-item-label'));
-      expect(items.length).toBe(2);
-      expect(items[0].children.length).toBe(2);
-      expect(items[0].children[0].nativeElement.classList).toContain('ant-segmented-item-icon');
-      expect(items[0].children[1].nativeElement.tagName).toBe('SPAN');
-      expect(items[1].children.length).toBe(1);
-      expect(items[1].children[0].nativeElement.classList).toContain('ant-segmented-item-icon');
+    it('should render only one element in .ant-segmented-item-label if the item is icon-only', async () => {
```

Comment:
I change this test cause, currently in the pipeline it failed

<img width="1264" height="706" alt="Screenshot 2025-08-30 at 09 39 45" src="https://github.com/user-attachments/assets/221b661b-5373-423d-b6cb-282ca9d08a2c" />

Now test pass :) 

"maybe there is an issue with the pipeline cause, i saw that some test failed but the job test of the pipeline is success